nltk/tokenize/moses.py

```diff
@@ -245,6 +245,7 @@ def __init__(self, lang='en'):
         self.NUMERIC_ONLY_PREFIXES = [w.rpartition(' ')[0] for w in
                                       self.NONBREAKING_PREFIXES if
                                       self.has_numeric_only(w)]
+        self.no_escaping = no_escaping
```

Comment:
If the parameter controlling the escaping is at the function then we wouldn't need the self.escape parameter too.